docs/reference/testing/strategies.rst

```diff
@@ -0,0 +1,80 @@
+.. py:module:: craft_platforms.test.strategies
+
+Hypothesis strategies
+=====================
+
+Craft Platforms provides several :external+hypothesis:doc:`Hypothesis <index>`
+strategies to assist with testing.
+
+Distributions and series
+------------------------
+
+The primary strategy here is to generate :py:class:`~craft_platforms.DistroBase`
+objects based on real Linux distributions:
+
+.. autofunction:: real_distro_base
+
+However, if you need to check that you can handle any realistic value, you can use:
+
+.. autofunction:: any_distro_base
+
+Specific distributions
+~~~~~~~~~~~~~~~~~~~~~~
+
+If you need to test against a specific distribution, the following strategies are
+available:
+
+.. autofunction:: debian_series
```

Comment:
Are these functions supposed to show up in the reference? I don't see them:

https://canonical-craft-platforms--107.com.readthedocs.build/en/107/reference/testing/strategies.html

Reply:
Fascinating. It renders correctly on my machine, so I'll have to look in more detail. 

![image](https://github.com/user-attachments/assets/65c40288-ea17-40f1-b954-915868ec7d19)